ion_hash_tests.ion

```diff
@@ -922,56 +908,6 @@ sexpCount128::{
              0x0e)
   ) }
 }
-{
-  ion:{               // verify various types get added to a struct
-    'null':null,
-    bool:false,
-    int:5,
-    neg_int:-6,
-    float:4.9e-324,
-    decimal:123.45,
-    timestamp:2017-01-01T00:00:00-00:00,
-    symbol:hello,
-    string:"hello",
-    clob:{{"hello"}},
-    blob:{{aGVsbG8=}},
-    list:[1,2,3],
-    sexp:(1 2 3),
-    struct:{},
-    annotated_value:hello::{},
```

Comment:
What is the reason that this test is failing? Which unsupported feature?

Reply:
This test has timestamps which is currently being worked on in Ion Go. Will need to rerun this once timestamp fix is in.

Added comment